system-x/services/infinispan/src/main/java/software/tnb/infinispan/service/Infinispan.java

```diff
@@ -21,6 +21,6 @@ public Map<String, String> containerEnvironment() {
 
     @Override
     public String defaultImage() {
-        return "registry.redhat.io/datagrid/datagrid-8-rhel8:latest";
+        return "registry.redhat.io/datagrid/datagrid-8-rhel9:latest";
```

Comment:
please, while you're at it, also change the tag to `1.5-8.5.1.GA`, so that we don't get unexpected bumps

Reply:
mm are you sure? I'd rather know if Camel is compatible with the latest version (so that we can document it in case), wdyt? Especially for Red Hat products